### PR TITLE
Phase 6: Combined Baseline 8-Seed Validation (aft_foil_srf + gap/stagger aug)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

The current single-model baseline (p_in=13.19, p_oodc=7.92, p_tan=30.05, p_re=6.45) comes from PR #2104 which validated `--aft_foil_srf` using 8 seeds **without** `--aug_gap_stagger_sigma 0.02`. PR #2115 then merged gap/stagger augmentation, validated separately against an earlier baseline (also without aft_foil_srf). The reproduce command now includes both flags, but the combined 8-seed mean has never been measured.

**Why this matters:** With 7 WIP experiments about to land, accurate baseline targets are critical for merge decisions. If the combined baseline has p_oodc ≈ 7.53 (applying the -4.9% gap/stagger gain additively), a PR showing p_oodc=7.6 would appear to beat 7.92 but actually represents a regression. This validation run gives us ground truth.

This is a **pure validation run** — no code changes. Run 8 seeds of the current reproduce command and report the 8-seed mean ± std.

## Instructions

**No code changes required.** Run the standard reproduce command for seeds 42–49 in parallel:

```bash
BASE="--agent alphonse --wandb_group phase6/combined-baseline-8seed \
  --asinh_pressure --asinh_scale 0.75 --field_decoder --adaln_output --use_lion --lr 2e-4 \
  --aug aoa_perturb --aug_full_dsdf_rot --high_p_clamp --n_layers 3 --slice_num 96 \
  --tandem_ramp --domain_layernorm --domain_velhead --ema_decay 0.999 --weight_decay 5e-5 \
  --cosine_T_max 160 --disable_pcgrad --pressure_first --pressure_deep \
  --residual_prediction --surface_refine --surface_refine_hidden 192 --surface_refine_layers 3 \
  --aft_foil_srf --aug_gap_stagger_sigma 0.02"

for seed in 42 43 44 45 46 47 48 49; do
  python train.py $BASE \
    --seed $seed \
    --wandb_name "alphonse/combined-baseline-s${seed}" &
done
wait
```

**What to report:** Table with columns (seed, p_in, p_oodc, p_tan, p_re, val/loss, W&B run ID) for all 8 seeds, plus 8-seed mean ± std. This becomes the official single-model baseline for all future merge decisions, replacing the current stale targets.

## Baseline

Previous single-model baselines for comparison:

| Config | p_in | p_oodc | p_tan | p_re |
|--------|------|--------|-------|------|
| aft_foil_srf only (PR #2104, 8-seed, seeds 42-49) | 13.19 ± 0.33 | 7.92 ± 0.17 | **30.05** ± 0.36 | 6.45 ± 0.07 |
| gap/stagger σ=0.02 (PR #2115, 2-seed, no aft_foil_srf) | ~13.04 | 7.446 | 30.213 | 6.351 |
| asinh-only baseline (PR #2054, 8-seed) | 13.03 | 7.83 | 30.29 | 6.45 |

**Reproduce command (current BASELINE.md):**
```bash
cd cfd_tandemfoil && python train.py --agent alphonse \
  --wandb_name "alphonse/combined-baseline-s42" --seed 42 \
  --asinh_pressure --asinh_scale 0.75 --field_decoder --adaln_output --use_lion --lr 2e-4 \
  --aug aoa_perturb --aug_full_dsdf_rot --high_p_clamp --n_layers 3 --slice_num 96 \
  --tandem_ramp --domain_layernorm --domain_velhead --ema_decay 0.999 --weight_decay 5e-5 \
  --cosine_T_max 160 --disable_pcgrad --pressure_first --pressure_deep \
  --residual_prediction --surface_refine --surface_refine_hidden 192 --surface_refine_layers 3 \
  --aft_foil_srf --aug_gap_stagger_sigma 0.02
```

### CODE REVIEW

**Comment by tcapelle** [comment]
## Experiment Results — Combined Baseline 8-Seed Validation

All 8 seeds completed at epoch 153-154 (180-min wall-clock timeout). W&B group: `phase6/combined-baseline-8seed`.

### Per-Seed Results

| Seed | p_in | p_oodc | p_tan | p_re | val/loss | W&B Run ID |
|------|------|--------|-------|------|----------|------------|
| 42 | **12.49** | **7.4** | 30.7 | 6.5 | 0.3878 | `zapen0x3` |
| 43 | 13.33 | 7.9 | 30.9 | 6.5 | 0.3918 | `3vuz3adi` |
| 44 | 13.57 | 7.9 | 30.8 | 6.6 | 0.3945 | `g1uhcorj` |
| 45 | 13.21 | 7.7 | 30.4 | 6.4 | 0.3891 | `ea551p6b` |
| 46 | 13.38 | 8.0 | 31.0 | 6.4 | 0.3944 | `fdf1vsi3` |
| 47 | 13.09 | 7.8 | **29.6** | 6.6 | 0.3850 | `al6opl9g` |
| 48 | 13.49 | **7.4** | 29.9 | 6.5 | 0.3859 | `jg15oow3` |
| 49 | 13.33 | 7.7 | 30.9 | 6.5 | 0.3897 | `vzm3s42y` |
| **Mean** | **13.24** | **7.73** | **30.53** | **6.50** | **0.3898** | — |
| **Std** | **0.33** | **0.22** | **0.50** | **0.07** | **0.0034** | — |

### Comparison with Previous Baselines

| Config | p_in | p_oodc | p_tan | p_re |
|--------|------|--------|-------|------|
| **Combined (this run)** | **13.24 ± 0.33** | **7.73 ± 0.22** | 30.53 ± 0.50 | **6.50 ± 0.07** |
| aft_foil_srf only (PR #2104, 8-seed) | 13.19 ± 0.33 | 7.92 ± 0.17 | **30.05 ± 0.36** | 6.45 ± 0.07 |
| gap/stagger σ=0.02 (PR #2115, 2-seed) | ~13.04 | 7.45 | 30.21 | 6.35 |
| asinh-only (PR #2054, 8-seed) | 13.03 | 7.83 | 30.29 | 6.45 |

### Analysis

The combined baseline with `--aft_foil_srf --aug_gap_stagger_sigma 0.02` yields:

1. **p_oodc: 7.73 (-2.4% vs aft_foil_srf-only 7.92)** — gap/stagger augmentation helps OOD-condition generalization as expected.
2. **p_in: 13.24 (≈ same as 13.19)** — no meaningful change on in-distribution.
3. **p_tan: 30.53 (+1.6% vs 30.05)** — slight regression on tandem transfer. This may reflect a noise floor rather than a real interaction, but it's worth noting.
4. **p_re: 6.50 (≈ same as 6.45)** — no meaningful change on Reynolds OOD.

**Key takeaway:** The combination is not simply additive. Gap/stagger augmentation's OOD benefit (p_oodc -2.4%) is partially offset by a small tandem transfer regression. The net effect is a modest improvement on OOD generalization with stable in-distribution and Reynolds performance.

**Significance thresholds (mean - 2σ):**
- p_in < 12.58, p_oodc < 7.29, p_tan < 29.53, p_re < 6.36

These become the official single-model baseline targets for future merge decisions.

### Suggested Follow-ups

- The p_tan regression (30.53 vs 30.05) warrants investigation. It could be that gap/stagger noise slightly disrupts the tandem transfer geometry encoding.
- Consider whether aft_foil_srf and gap/stagger aug interact negatively — the aft-foil SRF head may be sensitive to gap/stagger perturbations during training.

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2123 because ADVISOR: Closing — validation complete, baseline updated.

**Critical finding:** The combined baseline (aft_foil_srf + aug_gap_stagger σ=0.02) is NOT simply additive:
- p_oodc: 7.73 ± 0.22 (**-2.4% vs aft_srf-only 7.92** ✓)
- p_tan: 30.53 ± 0.50 (**+1.6% vs aft_srf-only 30.05** ✗)
- p_in: 13.24 ± 0.33 (~neutral)
- p_re: 6.50 ± 0.07 (~neutral)

Gap/stagger augmentation helps OOD-condition generalization but regresses our primary metric (p_tan). This is important context for all in-flight experiments running with both flags.

**Official single-model baseline is now updated** to these 8-seed combined numbers: p_in=13.24, p_oodc=7.73, p_tan=30.53, p_re=6.50. All merge decisions will compare against these targets.

Excellent work running this validation — it caught a masked regression that would have led to incorrect merge decisions. Will assign you a new experiment shortly..
